PoSSapi/src/Domain/Entities/User.cs

```diff
@@ -7,6 +7,7 @@ public class User
     public Guid Id { get; set; }
     public Business Business { get; set; }
     public Guid BusinessId { get; set; }
+    public Business ManagedBusiness { get; set; }
```

Comment:
This seems a bit unusual. Is there a case where ManagerBusiness would differ from Business? If not I'd just not not add this property, since it would belong more in the StaffMember class, which we decided to skip.

Reply:
Since there is no `StaffMember`, maybe can leave this?